website/src/components/ExecutionQueue/exeQueue.css

```diff
@@ -0,0 +1,9 @@
+.queue__data{
+    justify-content: space-around;
```

Comment:
Let's just add a header on this file and on: `PreviousExeRes.jsx`, `previousExeRes.css`